lib/types.ts

```diff
@@ -191,12 +197,8 @@ export interface EditorInterface {
     settings?: Object
     disabled?: boolean
   }>
-  editor?: {
-    widgetId: string
-    widgetNamespace: string
-    settings?: Object
-  }
-  editors?: Array<EditorInterface['editor']>
```

Comment:
This code was making `EditorInterface['editor']` optional since it is an optional property.